### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by hatevadym**
- [DEMO LINK](https://hatevadym.github.io/layout_catalog/)
- [TEST REPORT LINK](https://hatevadym.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ Make all the changes smooth on hover (during 300ms):
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://<hatevadym>.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://<hatevadym>.github.io/layout_catalog/report/html_report/)
```
Comment (DarkMistyRoom) [inline readme.md:34]:
```suggestion
- [DEMO LINK](https://hatevadym.github.io/layout_catalog/)
- [TEST REPORT LINK](https://hatevadym.github.io/layout_catalog/report/html_report/)
```

Comment (DarkMistyRoom) [reply readme.md:34]: Your demo links don't work. Please, fix them and provide into the PR description

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:34]: create a separate file for this block

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:22]:
```suggestion
.container {
  padding: 50px 40px;
  display: grid;
  grid-template-columns: repeat(1, 200px);
  gap: 46px 48px;
  justify-content: center;
  
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }
}
```

Comment (DarkMistyRoom) [reply src/styles/main.scss:22]: you can do it this way

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      border: 1px solid $buttoncolor;
+      background-color: white;
+      color: $buttoncolor;
+    }
+  }
+}
+
+.card:hover .card__name {
+  color: $cardtexthover;
+}
```
Comment (DarkMistyRoom) [inline src/styles/blocks/card.scss:113]:
```suggestion
&:hover &__name {
  color: $cardtexthover;
}
```

Comment (DarkMistyRoom) [reply src/styles/blocks/card.scss:113]: move it into the block

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls fix prev comments
Pls ask for help in **fe_chat** if you don't understand something

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,306 @@
...
+              href="#video"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
```
Comment (volodymyr-soltys97) [inline src/index.html:112]:
```suggestion
    </header>
    
    <main class="container">
```

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,306 @@
...
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__pic"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:117]: This image isn't background, so you need to add image with the `img` tag

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,306 @@
...
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:121]: You need to use the semantic tag for text, fix it everywhere
```suggestion
        <h2 class="card__name">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
        
        <p class="card__code">Product code: 195434</p>
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, the demo link hasn't styles, need to fix it 
image

**Pull request closed without merging**